modules/images/webp-uploads/load.php

```diff
@@ -157,6 +158,41 @@ function webp_uploads_create_sources_property( array $metadata, $attachment_id )
 
 add_filter( 'wp_generate_attachment_metadata', 'webp_uploads_create_sources_property', 10, 2 );
```

Comment:
See above, I guess I missed that before. Not critical in any case.
```suggestion
add_filter( 'wp_generate_attachment_metadata', 'webp_uploads_create_sources_property', 10, 2 );
```

Reply:
Updated.